### PR TITLE
kim: Add version 0.1.0-beta.6

### PR DESCRIPTION
**Pull request by reicheltp**
https://github.com/rancher/kim - The Kubernetes Image Manager

### CODE REVIEW

File: bucket/kim.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://github.com/rancher/kim",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/rancher/kim/releases/download/v0.1.0-beta.6/kim-windows-amd64.exe#/kim.exe",
+            "hash": "64fe06e075fb9cf5e5116bff1e7dc7154c0e2f14ff112f3a352ed1ea97987b0f"
+        }
+    },
+    "bin": "kim.exe",
+    "checkver": "github",
```
Comment (issaclin32) [inline bucket/kim.json:13]:
```suggestion
    "checkver": {
        "github": "https://github.com/rancher/kim",
        "regex": "download/v([\\w.-]+)/kim-windows-amd64.exe"
    },
```

Comment (issaclin32) [reply bucket/kim.json:13]: `"checkver": "github"` will not work because *Scoop* matches `([\\d.]+)` by default.

Comment (reicheltp) [reply bucket/kim.json]: > `"checkver": "github"` will not work because *Scoop* matches `([\\d.]+)` by default.

Thanks!

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**